LayoutTests/platform/ios/TestExpectations

```diff
@@ -3719,4 +3719,8 @@ imported/w3c/web-platform-tests/css/css-ui/compute-kind-widget-generated/kind-of
 imported/w3c/web-platform-tests/css/css-ui/compute-kind-widget-generated/kind-of-widget-fallback-textarea-border-right-color-001.html [ Pass ImageOnlyFailure ]
 imported/w3c/web-platform-tests/css/css-ui/compute-kind-widget-generated/kind-of-widget-fallback-textarea-border-top-color-001.html [ Pass ImageOnlyFailure ]
 
+# iOS user agent styles are preventing these from passing for <input type=search>
+imported/w3c/web-platform-tests/css/css-ui/appearance-textfield-001.html [ ImageOnlyFailure ]
```

Comment:
Can you file a bug to track these failures?

Reply:
https://bugs.webkit.org/show_bug.cgi?id=241095